CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+**Added**
+
+- SecureNN with int64 support has landed.
```

Comment:
maybe mark somewhere that a nightly/custom version of TF is needed for int64 (but CRT works in either case)

Reply:
O yeah. Thanks.